charts/thoras/README.md

```diff
@@ -46,112 +49,118 @@ helm install \
 # Values
 
 ## Global
-| Key | Type | Default | Description |
-| --- | --- | --- | --- |
-| thorasVersion | String | 1.1.5 | Thoras app version |
-| imageCredentials.registry | String | us-east4-docker.pkg.dev/thoras-registry/platform | Container registry name |
-| imageCredentials.username | String | _json_key_base64 | Container registry username |
-| imageCredentials.password | String | "" | Container registry auth string |
-| resourceQuota.enabled | Bool | false | Enables resource quotas within Thoras |
-| resourceQuota.pods | Number | 200 | Maximum number of pods allowed|
-| resourceQuota.cronjobs | Number | 200 | Maximum number of cronjobs allowed |
-| resourceQuota.jobs | Number | 200 | Maximum number of jobs allowed |
-| logLevel | String | info | Default log level |
-| slackWebhookUrl | String | "" | Slack Webhook URL destination for notifications. |
-| slackErrorsEnabled | Boolean | false | Determines if error-level logs are sent to `slackWebHookUrl` |
+
+| Key                       | Type    | Default                                          | Description                                                  |
+| ------------------------- | ------- | ------------------------------------------------ | ------------------------------------------------------------ |
+| thorasVersion             | String  | 1.1.5                                            | Thoras app version                                           |
+| imageCredentials.registry | String  | us-east4-docker.pkg.dev/thoras-registry/platform | Container registry name                                      |
+| imageCredentials.username | String  | \_json_key_base64                                | Container registry username                                  |
```

Comment:
[dust] What's the additional `\` for? Was it not rendering correctly before without the escape?

Reply:
I'll be honest with you.. 

Prettier automatically added these and I didnt even notice

Reply:
It seems to render either way.